dash-renderer/digest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "MD5 (dash_renderer.dev.js)":"e6d5a2bb6733ecd334d8aa4369560afe",
+    "MD5 (dash_renderer.dev.js.map)":"2f85eb3e5a9259ec6809513796e5f4aa",
+    "MD5 (dash_renderer.min.js)":"6904f54f3f135ae5e66853ce9e7e7bec",
+    "MD5 (dash_renderer.min.js.map)":"97e13dc663efea82b06083cab8520c1c",
+    "MD5 (prop-types@15.7.2.js)":"85947944e396a28895fad5f553eee36f",
+    "MD5 (prop-types@15.7.2.min.js)":"e3053393609bd2744010498629a43597",
+    "MD5 (react-dom@16.8.6.js)":"d7f8afaf3370a228c8d5c802c9d9a102",
+    "MD5 (react-dom@16.8.6.min.js)":"fad5842bd019c3878795ec52059f47fc",
+    "MD5 (react@16.8.6.js)":"bb95f4cd851114c374c3858e9c51da10",
+    "MD5 (react@16.8.6.min.js)":"f808b8e8ab51b0d9525795db3768cd86",
+    "dash_renderer":"1.0.0"
+}
```

Comment:
Not sure I understand how this works. Why do we keep this build artifact but remove all the others?

Reply:
Do you mean removing all the dep bundles?

Reply:
Yes. But I realize now that this is pointing to `dev` and that that branch already doesn't have the other build artifacts.

Reply:
Would include this file in the `.gitignore` for `dev` but keep it around in `master`